podcasts/ChapterManager.swift

```diff
@@ -12,7 +12,7 @@ class ChapterManager {
 
     private var lastEpisodeUuid = ""
 
-    var currentChapters = Chapters(chapters: [])
+    var currentChapters: Chapters = Chapters()
```

Comment:
We can remove the Chapters type here since it's inferred from the `Chapters()` init.

```suggestion
    var currentChapters = Chapters()
```

Reply:
Removed type specification